cider-velox/src/CiderHashJoinBuild.h

```diff
@@ -44,7 +44,7 @@ class CiderHashJoinBridge : public exec::JoinBridge {
   std::optional<CiderHashBuildResult> hashBuildResultOrFuture(ContinueFuture* future);
 
  private:
-  std::optional<cider::processor::HashBuildResult> buildResult_;
+  std::optional<cider::exec::processor::HashBuildResult> buildResult_;
 };
 
 class CiderHashJoinBuild : public exec::Operator {
```

Comment:
rename to `CiderHashJoinBuildOperator`?

Reply:
Just follow the name convention in velox